winmd.go

```diff
@@ -71,10 +71,17 @@ type CodedIndex struct {
 //
 // It is used as an optimization to avoid allocating
 // when reading from the #Strings heap.
-type String []byte
+type String struct {
```

Comment:
A good side effect of this change is that `String` no longer exposes the backing array, so we have a proper inmutable structure.